src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd.rej

```diff
@@ -0,0 +1,13 @@
+diff a/src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd b/src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd	(rejected hunks)
+@@ -6,9 +6,10 @@
+ set TOOLRUNTIME_DIR=%~3
+ IF [%BUILDTOOLS_TARGET_RUNTIME%]==[] set BUILDTOOLS_TARGET_RUNTIME=win7-x64
+ set BUILDTOOLS_PACKAGE_DIR=%~dp0
++set MICROBUILD_VERSION=0.2.0
+ set PORTABLETARGETS_VERSION=0.1.1-dev
+ set ROSLYNCOMPILERS_VERSION=1.2.0-beta1-20160202-02
+-set MSBUILD_CONTENT_JSON={"dependencies": {"Microsoft.Portable.Targets": "%PORTABLETARGETS_VERSION%", "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%"},"frameworks": {"dnxcore50": {},"net46": {}}}
++set MSBUILD_CONTENT_JSON={"dependencies": { "MicroBuild.Core": "%MICROBUILD_VERSION%", Microsoft.Portable.Targets": "%PORTABLETARGETS_VERSION%", "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%"},"frameworks": {"dnxcore50": {},"net46": {}}}
```

Comment:
There is a missing " here, @ericstj checked in a fix for it earlier today so be sure to pick it up.